src_c/mixer.c

```diff
@@ -725,6 +725,9 @@ snd_get_volume(PyObject *self, PyObject *_null)
     MIXER_INIT_CHECK();
 
     volume = Mix_VolumeChunk(chunk, -1);
+    if (volume == -1) {
+        return PyLong_FromLong(volume);
+    }
```

Comment:
Hmm... `Mix_VolumeChunk` returns `-1` when `chunk` is `NULL`. I feel like that's the real problem here. Why is a python `Sound` class associated with a `NULL` `Mix_Chunk`?

It'd also be nice to not forward things like `-1`s into python land. It's neater to raise python exceptions

Reply:
I see what you mean, guess I've been using low-level languages too long these days. I feel then this PR is not needed, but I reopened a PR that does these checks.